tokio/src/time/delay.rs

```diff
@@ -33,6 +33,24 @@ pub fn delay_until(deadline: Instant) -> Delay {
 ///
 /// Canceling a delay is done by dropping the returned future. No additional
 /// cleanup work is required.
+///
+/// # Examples
+///
+/// Wait 100ms and print "100 ms have elapsed". Note that no work is
+/// performed while awaiting on the delay to complete. If you wish to do
+/// other work while awaiting please see [`interval`].
```

Comment:
Can you change it so it says that `interval` is used for regularly running something according to a schedule or something like that? And move the part about `interval` up before the `# Cancellation` header?

Reply:
How is the following? 
> If you wish to run something regularly on a schedule see [`interval`].

Reply:
I prefer to avoid "If you wish" in documentation, but other than that it is fine

> To run something regularly on a schedule, see `interval`.

Reply:
Okay! Thanks.